core/state_processor.go

```diff
@@ -249,7 +249,7 @@ func ApplyTransactionOnMPS(config *params.ChainConfig, bc *BlockChain, author *c
 
 // /Quorum
 
-func applyTransaction(msg types.Message, config *params.ChainConfig, bc *BlockChain, author *common.Address, gp *GasPool, statedb, privateStateDB *state.StateDB, header *types.Header, tx *types.Transaction, usedGas *uint64, evm *vm.EVM, forceNonParty bool) (*types.Receipt, *types.Receipt, error) {
+func applyTransaction(msg types.Message, config *params.ChainConfig, bc *BlockChain, gp *GasPool, statedb, privateStateDB *state.StateDB, header *types.Header, tx *types.Transaction, usedGas *uint64, evm *vm.EVM) (*types.Receipt, *types.Receipt, error) {
```

Comment:
This function is a refactorisation of geth, it has some parameters (author and forceNonParty) that are not used and I removed them.